include/agl/driver/aglNVNMgr.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <heap/seadDisposer.h>
+#include <hostio/seadHostIONode.h>
+#include "agl/driver/aglGraphicsDriverMgr.h"
+
+namespace agl::driver {
+
+class NVNMgr : GraphicsDriverMgr {
+public:
+    // TODO: This is wrong and should actually touch the GraphicsDriverMgr implementation
+    SEAD_SINGLETON_DISPOSER(NVNMgr)
```

Comment:
This should go before the `public:` to make sure that any access specifiers that are in the SINGLETON macro do not affect anything you declare below